frontend/src/serviceWorker.js

```diff
@@ -29,7 +29,7 @@ export function register(config) {
             return;
         }
 
-        window.addEventListener('load', () => {
+        window.addEventListener('load', async () => {
```

Comment:
I think this file is generated by create-react-app, so we shouldn't have to modify it -- let's leave it as is.